src/pages/Home.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import '../App.css';
+import NewListButton from '../components/NewListButton';
+import * as ls from 'local-storage';
+/*import FirebaseTesting from './FirebaseTest';*/
+import fb from '../lib/firebase';
+import { Link, Redirect } from 'react-router-dom';
+import FooterTabs from '../components/FooterTabs';
+/*import ShoppingList from './ShoppingList';
+import AddItem from './AddItem';*/
+
+function Home() {
```

Comment:
```suggestion
function Home({token, setToken}) {
```
Since we are sending token and setToken as props, we can unpack the object into token and setToken. This way the NewListButton component on line 52 still has access to token and setToken passed from App, through Home.